### ISSUE TITLE
dev-qt/qtnetwork-5.6.1 Failed to emerge 

### ISSUE THREAD

**Issue by kittkott** [gentoo/libressl#159]
```
>>> Fetching (1 of 1) dev-qt/qtnetwork-5.6.1::libressl
!!! Fetched file: qtbase-opensource-src-5.6.1-1.tar.xz VERIFY FAILED!
!!! Reason: Insufficient data for checksum verification
!!! Got:
!!! Expected: MD5 RMD160 SHA1 SHA256 SHA512 WHIRLPOOL
 * Fetch failed for 'dev-qt/qtnetwork-5.6.1'

```

**Comment by csmk** [comment]
The PR #160 should fix the issue.

**Comment by kittkott** [comment]
The download file name is qtbase-opensource-src-5.6.1-1.tar.xz
In "libressl/dev-qt/qtnetwork/Manifest" should be corrected to 

`DIST qtbase-opensource-src-5.6.1-1.tar.xz 46788436 SHA256 329678347ec5ebb404225345300a8deb1e7c991322a4c50584be550c69be7c39 SHA512 e9660bee2e0acd91c3232831e4f1a99ad30b82b8ff26850494efec89a525499b721adfb63f7ede586408489d3ebf727b842cd12eab665c58769001454e7f661d WHIRLPOOL e26cf72d9644dbcb08d1269bdaa633be2e24de813deb62a1c77e448b6578f6761769664fc95c4131c342f077c092a777af8fed62ea35e5fdb0f2576d82b3c499`

After that 
```
emerge =dev-qt/qtnetwork-5.6.1 -f
>>> Fetching (1 of 1) dev-qt/qtnetwork-5.6.1::libressl
 * qtbase-opensource-src-5.6.1-1.tar.xz SHA256 SHA512 WHIRLPOOL size ;-) ...                                                                          [ ok ]

```
And emerge is OK. 

```
emerge =dev-qt/qtnetwork-5.6.1 -av
[ebuild   R   ] dev-qt/qtnetwork-5.6.1 [5.6.1] USE="libressl%* networkmanager ssl -bindist -connman -debug -libproxy {-test}"

>>> Verifying ebuild manifests
>>> Emerging (1 of 1) dev-qt/qtnetwork-5.6.1::libressl
>>> Installing (1 of 1) dev-qt/qtnetwork-5.6.1::libressl
>>> Jobs: 1 of 1 complete

```

**Comment by blueness** [comment]
I just merged pr #160.  can you let me know if this is fixed.

**Comment by kittkott** [comment]
Yea, issue is fixed.
